docs/crio.conf.5.md

```diff
@@ -125,7 +125,7 @@ The `crio.runtime` table contains settings pertaining to the OCI runtime used an
   Additional environment variables to set for all the containers. These are overridden if set in the container image spec or in
 the container runtime configuration.
 
-**selinux**=false
+**selinux**=true
```

Comment:
oops this seems to have slipped in

Reply:
make docs-validation fails with 
```go run -tags "containers_image_ostree_stub      containers_image_openpgp seccomp selinux " ./test/docs-va[5/2200]
INFO Verifying TOML tags of `config.go` to `TemplateString` and `docs/crio.conf.5.md`                            
ERRO Tag `selinux` with expected value `true` not found in `docs/crio.conf.5.md` ``` 
if it's not sent to true :thinking: